app/views/child_benefit_tax/main.html+new_navigation.erb

```diff
@@ -0,0 +1,138 @@
+<% content_for :title, "Child Benefit tax calculator" %>
+<% content_for :body_classes, "full-width" %>
+
+<div class="grid-row">
+  <main id="content" role="main" class="group calculator-page">
```

Comment:
(We can reassess after Template Consolidation merge [`to-elements`](https://github.com/alphagov/calculators/tree/to-elements))